app/javascript/app/pages/about/about.js

```diff
@@ -0,0 +1,160 @@
+import { createElement } from 'react';
+
+import ndcImage from 'assets/partners/ndcp.png';
+import wriImage from 'assets/partners/wri.png';
+import climateAnalyticsImage from 'assets/partners/climate-analytics.png';
+import climateActionTrackerImage from 'assets/partners/climate-action-tracker.png';
+import worldBankImage from 'assets/partners/the-world-bank.png';
+import ccImage from 'assets/partners/cc.png';
+import gizImage from 'assets/partners/giz.png';
+import googleImage from 'assets/partners/google.png';
+import jgcriImage from 'assets/partners/jgcri.png';
+import fmencbnsImage from 'assets/partners/fmencbns.png';
+import dbeisImage from 'assets/partners/dbeis.png';
+import swissImage from 'assets/partners/swiss.png';
+
+import Component from './about-component';
+
+const jointInitiative = {
```

Comment:
I can't remember what we decided about constants like this. Whether to keep them in initial state or in the container file?